orca-core/src/main/groovy/com/netflix/spinnaker/orca/jackson/OrcaObjectMapper.groovy

```diff
@@ -17,24 +17,18 @@
 package com.netflix.spinnaker.orca.jackson
 
 import com.fasterxml.jackson.databind.ObjectMapper
-import com.fasterxml.jackson.databind.module.SimpleModule
 import com.fasterxml.jackson.datatype.guava.GuavaModule
 import com.fasterxml.jackson.datatype.jdk8.Jdk8Module
-import com.netflix.spinnaker.orca.pipeline.model.Stage
 import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL
 import static com.fasterxml.jackson.databind.DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES
 
 class OrcaObjectMapper extends ObjectMapper {
```

Comment:
not something to change in the scope of this PR but subclassing ObjectMapper is always a bad idea (we should have done a factory thingy)